src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -50,9 +51,10 @@ def get(self, request: Request, organization: Organization) -> Response:
         except NoProjects:
             return Response({})
 
+        data_source = Dataset.IssuePlatform if request.GET.get("data_source") else Dataset.Discover
```

Comment:
why get rid of the validator? now it doesn't check at all if the parameter is correct before setting the data source